src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -258,6 +264,25 @@ private InitCodeView buildInitCodeViewRequestObject(
     return paths.stream().map(path -> root.subTree(path)).collect(ImmutableList.toImmutableList());
   }
 
+  @VisibleForTesting
+  static void assertNoOverlap(InitCodeNode root, List<String> paths) {
```

Comment:
Could you make the function signature more descriptive?
eg `assertNoOverlap(newNode, previousPaths)`
or `assertArgumentDistinct(...)`

Reply:
I don't think `newNode` and `prevPaths` are good names, since the function acts on all paths at once; there's no before or after here.

Reply:
OK. I guess I was thinking that since you call it every time you add a new param to the view.  (Is that right?)

Reply:
I don't think so? This is called only once with all the params.